var/spack/repos/builtin/packages/adios/package.py

```diff
@@ -134,7 +150,15 @@ def configure_args(self):
             extra_args.append('--with-szip=%s' % spec['szip'].prefix)
         if '+zfp' in spec:
             extra_args.append('--with-zfp=%s' % spec['zfp'].prefix)
+        if '+sz' in spec:
+            extra_args.append('--with-sz=%s' % spec['sz'].prefix)
         if '+hdf5' in spec:
             extra_args.append('--with-phdf5=%s' % spec['hdf5'].prefix)
+        if '+netcdf' in spec:
+            extra_args.append('--with-netcdf=%s' % spec['netcdf'].prefix)
+        if ('+flexpath' in spec) or ('+staging' in spec):
+            extra_args.append('--with-flexpath=%s' % spec['libevpath'].prefix)
+        if ('+dataspaces' in spec) or ('+staging' in spec):
+            extra_args.append('--with-dataspaces=%s' % spec['dataspaces'].prefix)
```

Comment:
It would be good to add:
```python
else:
    extra_args.append('--without-<feature>')
```
for all of these if-statements. Otherwise I'm afraid it might still build with NetCDF if it finds it installed on your system.